app/assets/javascripts/app/views/aspect_membership_view.js

```diff
@@ -13,8 +13,7 @@ app.views.AspectMembership = app.views.AspectsDropdown.extend({
 
   events: {
     "click ul.aspect_membership.dropdown-menu > li.aspect_selector": "_clickHandler",
-    "keypress ul.aspect_membership.dropdown-menu > li.aspect_selector": "_clickHandler",
-    "click ul.aspect_membership.dropdown-menu > li.newItem": "showModal"
+    "keypress ul.aspect_membership.dropdown-menu > li.aspect_selector": "_clickHandler"
```

Comment:
I'm not sure there's much to discuss, I think you'll have a hard time to find actual good examples for it, since backbone views are generally meant to be reusable.